RetrospectiveExtension.Frontend/components/feedbackCarousel.tsx

```diff
@@ -24,7 +26,7 @@ export default class FeedbackCarousel extends React.Component<IFeedbackCarouselP
       .filter((columnItem) => !columnItem.feedbackItem.parentFeedbackItemId)
       .map((columnItem) => {
         const feedbackItemProps =
-          FeedbackColumn.createFeedbackItemProps(feedbackColumnProps, columnItem, true);
+        FeedbackColumnStatic.createFeedbackItemProps(feedbackColumnProps, columnItem, true);
```

Comment:
I'd move `createFeedbackItemProps` static function to a utility class and use it, instead of exporting same class with two different approaches (withAITracking and with static)

it seems `createFeedbackItemProps` method is the only method we need here

Reply:
good point. I will create a utility class and name `FeedbackColumnHelperClass`